### PR TITLE
Added units like Hz, Pa and J to the mks unitsystem.

### PR DESCRIPTION
**Pull request by paramsingh**
Some units had been overlooked in mks.py and were missing. Added them to
the mks unitsystem. Fixes issue #8880.

### CODE REVIEW

**Comment by vramana** [comment]
@paramsingh I would have added the line you have added before the for loop starts. I may be wrong but I think the problem is with the way `prefix_unit` function is defined and used. @asmeurer What should be the correct way to fix this issue?

**Comment by paramsingh** [comment]
@vramana I could modify the `prefix_unit` function to include the original unit in the list it returns too, although I don't know if that would make any sense.

**Comment by smichr** [comment]
:information_source: could you please change the title of the PR? Including the issue number there isn't as useful as putting it in the Description (first paragraph above) because it doesn't create a clickable link in the Title.

:eyeglasses: See also: **Writing pull request title and description** at [the wiki](https://github.com/sympy/sympy/wiki/Development-workflow)

**Comment by paramsingh** [comment]
@smichr Sorry about that. Fixed.

**Comment by paramsingh** [comment]
Rebased and moved the line to before the for loop.

@vramana, how about now?

File: sympy/physics/unitsystems/systems/mks.py [hunk 1]
```diff
@@ -61,9 +61,16 @@
...
+
+# Prefixes of units like g, J, N etc get added using `prefix_unit`
+# in the for loop, but the actual units have to added manually.
+all_units.extend([g, J, N, W, Pa, Hz])
+
 for u in units:
     all_units.extend(prefix_unit(u, PREFIXES))
 all_units.extend([v, a, p, G, c])
 
+
```
Comment (vramana) [inline sympy/physics/unitsystems/systems/mks.py]: Remove the empty lines.

**Comment by paramsingh** [comment]
Done. Sorry about that, don't know how I missed it.

**Comment by paramsingh** [comment]
@smichr Is this okay or is this supposed to be fixed in some other way?

**Comment by kaichogami** [comment]
@paramsingh you should squash the commits and write a proper [commit message](https://github.com/sympy/sympy/wiki/Development-workflow#writing-commit-messages).

**Comment by vramana** [comment]
@paramsingh Do you know how to squash commits?

**Comment by paramsingh** [comment]
Done.

**Comment by smichr** [comment]
> Is this okay

Looks good to me. Thanks.

**Comment by paramsingh** [comment]
"to added" -> "to be added"

**Comment by paramsingh** [comment]
Huh, why did the build fail?

**Comment by smichr** [comment]
> fail

A Theano run didn't finish. It's all ok.

**Comment by smichr** [comment]
Thanks, this is in. And welcome to SymPy! :-)

:warning: the issue wasn't closed automatically because the format didn't follow the auto-close format. In the opening paragraph, "Fix issue #8880" should be "Fix #8880". I will close it by hand.

**Pull request merged**